daemon/api.go

```diff
@@ -958,7 +959,34 @@ func changeInterfaces(c *Command, r *http.Request) Response {
 		if len(a.Plugs) == 0 {
 			return BadRequest("at least one plug is required")
 		}
-		err := c.d.interfaces.AddPlug(&a.Plugs[0])
+		// NOTE: This constructs a partial snap.yaml meta-data. Later on it
+		// should reference real meta-data so that it chimes in with real
+		// snaps. Alternatively, this action should be removed.
```

Comment:
This was to maintain the illusion that they still work and do stuff but TBH I'm inclined to kill them later this week.